sokoban/src/player.ts

```diff
@@ -0,0 +1,31 @@
+export function handleBoard(e: any, newGameBoard: any, player: any, x: any, y: any, target: any) {
+  let copy = [...newGameBoard];
+  let newX = x;
+  let newY = y;
+
+  switch (e.key) {
+    case "ArrowUp":
+      newX = x - 1;
+      break;
+    case "ArrowDown":
+      newX = x + 1;
+      break;
+    case "ArrowLeft":
+      newY = y - 1;
+      break;
```

Comment:
arrow left should be x -1 arrow right should be x +1 , or the opposite :/

Reply:
If you push the arrow down then the player goes up to you?

Reply:
This is the playermovement function, not this one. But if you push the arrow down button goes the player up?